addons/mail/controllers/discuss/rtc.py

```diff
@@ -39,7 +39,7 @@ def session_call_notify(self, peer_notifications):
     @http.route("/mail/rtc/session/update_and_broadcast", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
     def session_update_and_broadcast(self, session_id, values):
-        """Update a RTC session and broadcasts the changes to the members of its channel,
+        """Update a RTC session and broadcasts the changes to the members of their channel,
```

Comment:
Isn't it the channel of the rtc session in this case?

Reply:
Yes it its, fixed.